qentaceecheckoutpage/upgrade/install-1.3.6.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
this file is not needed anymore, please note refactoring of it for the sustainment phase